package.json

```diff
@@ -26,7 +26,7 @@
   "version": "1.0.18",
   "private": false,
   "engines": {
-    "node": "v7.5.0"
+    "node": ">=7.5.0"
```

Comment:
Changed this otherwise it uses a very deprecated version of node, not compatible with npm ci builtin for example.
Let me now if that is ok

Reply:
this could be changed to node v10 at this point.

Reply:
Ok to keep as is for now

Reply:
I'm fine to set it as 10.21.0 as this is the default version I'm using in the CI for v10.
However what impact does that have when istalled as module, obviously none considering the current usage ?

In python there is the equivalent, and it would refuse du to mismatch